Tutorials/Extension_topics/Tuto_A.1_Parameter_estimation_for_compact_object_mergers_with_LALInference.ipynb

```diff
@@ -224,13 +229,13 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Extracting all files from /tmp/astropy-download-649822-pzjrf728\n"
+      "Extracting all files from /var/folders/54/s302xgds5h9f58fhfjzj7nm80000gp/T/astropy-download-24567-2m37l2y9\n"
      ]
     },
     {
      "data": {
       "text/plain": [
-       "PosixPath('/home/jerry/GW/odw-2024/Tutorials/Extension_topics/tuto_A.1/GW190828_063405')"
+       "PosixPath('/Users/mdubois/Projects/LVK/GWOSC/ODW/odw/Tutorials/Extension_topics/tuto_A.1/GW190828_063405')"
```

Comment:
Do we want to leave these specific home folders here? I think maybe we can leave the cell empty, without output if we these paths are crucial to the notebook.